cli/lib/models/raw_class.dart

```diff
@@ -0,0 +1,21 @@
+import 'item_class.dart';
+
+class RawItem extends ItemsClass {
+  double itemTax = 0.0;
+  double itemFinalPrice = 0.0;
+  RawItem(String? name, String? type, int quantity, double price)
+      : super(name, type, quantity, price);
+  @override
+  double calculateTax() {
+    print('calculating tax');
+    double tax = 12.5 / 100 * super.itemPrice;
+    itemTax = tax;
+    return itemTax;
+  }
+
+  @override
+  double calculatePrice() {
+    itemFinalPrice = super.itemPrice + itemTax;
+    return itemFinalPrice;
+  }
+}
```

Comment:
Convert all the hard-coded values like 12.5 into constants and give them meaningful names so that it is understandable what they represent